frame/composable-traits/src/staking/mod.rs

```diff
@@ -106,16 +204,16 @@ pub trait Staking {
 	/// * `to` the account to transfer the final claimed rewards to.
 	fn unstake(
 		who: &Self::AccountId,
-		instance_id: &Self::PositionId,
+		position: &Self::PositionId,
 		remove_amount: Self::Balance,
 	) -> DispatchResult;
 
-	/// `ratio` - how much of share to retain in original position.
+	/// `ratio` - how much of share to retain in the original position.
 	fn split(
 		who: &Self::AccountId,
-		instance_id: &Self::PositionId,
+		position: &Self::PositionId,
 		ratio: Permill,
```

Comment:
in math, it was 1/2, 1/3, 1/4. So it could not define all possible percentages.  When using usually stuff like (1-p) where where 0<=p<=1.  iam ok with current ration so.

Reply:
Yeah the ratio is not meant to be above 1. Regardless all we care about is that is a way to define the ratio for splitting

Reply:
but math was different. if was easy to make math and code be same.